src/main/resources/alfresco/public-services-context.xml

```diff
@@ -87,6 +88,13 @@
         </property>
     </bean>
 
+    <bean id="contentPropertyRestrictionInterceptor" class="org.alfresco.repo.node.ContentPropertyRestrictionInterceptor">
+        <property name="dictionaryService" ref="dictionaryService" />
+        <property name="nodeService" ref="mlAwareNodeService" />
+        <property name="globalContentPropertyRestrictions" value="${contentPropertyRestrictions.enabled}" />
+        <property name="globalContentPropertyRestrictionWhiteList" value="{contentPropertyRestrictions.white.list}"/>
```

Comment:
Probably a better name would be "whitelist" or "white-list". The dot however indicates some form of logical nesting.